src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -448,6 +462,8 @@ class FluxPipeline : public DiffusionPipeline {
                         ov::Tensor initial_image,
                         ov::Tensor mask_image,
                         const ov::AnyMap& properties) override {
+        const auto gen_start = std::chrono::steady_clock::now();
+        m_perf_metrics.clean_up();
```

Comment:
will it clean load time ?

Reply:
It does, so another variable is needed to save load time or `clean_up` does not clean load time. Which of these two methods is better?